trailblazer/constants.py

```diff
@@ -65,11 +59,20 @@ class SlurmJobStatus(str, Enum):
 
     CANCELLED: str = "cancelled"
     COMPLETED: str = "completed"
+    COMPLETING: str = "completing"
```

Comment:
what is the difference between "completing" and "running"

Reply:
It's one of the slurm status (https://slurm.schedmd.com/squeue.html#SECTION_JOB-STATE-CODES): 
 - COMPLETING: Job is in the process of completing. Some processes on some nodes may still be active
 - RUNNING: Job currently has an allocation